types/JestHooks.js

```diff
@@ -27,5 +30,8 @@ export type JestHookSubscriber = {
 export type JestHookEmitter = {
   onFileChange: (fs: JestHookExposedFS) => void,
   onTestRunComplete: (results: AggregatedResult) => void,
-  shouldRunTestSuite: (testPath: string) => Promise<boolean>,
+  shouldRunTestSuite: (
+    testPath: string,
+    config: ProjectConfig,
```

Comment:
Yeah, from the discussion above I could see the type here being `TestSuiteConfig` which is some subset of `Test`